com.onepassword.OnePassword.yml

```diff
@@ -0,0 +1,54 @@
+app-id: org.onepassword.onepassword
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: 1password
+rename-desktop-file: 1password.desktop
+rename-icon: 1password
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --socket=wayland
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-config
+  - --device=all
+modules:
+  - name: 1password
+    buildsystem: simple
+    build-commands:
+      - tar -xf 1password-latest.tar.gz
+      - mkdir -p /app/share/icons
+      - mkdir -p /app/1Password
+      - install -Dt /app/share/applications 1password-8.10.8.x64/resources/1password.desktop
+      - sed -i 's/Exec=.*/Exec=1password %U/' /app/share/applications/1password.desktop
+      - cp -a 1password-8.10.8.x64/resources/icons /app/share
+      - mv 1password.png /app/share/icons/hicolor/256x256/apps
+      - cp -a 1password-8.10.8.x64/* /app/1Password
+      - install -Dm755 1password.sh /app/bin/1password
+      - install -Dm644 com.onepassword.OnePassword.appdata.xml /app/share/appdata/com.onepassword.OnePassword.appdata.xml
+    sources:
+      - type: file
+        path: 1password.png
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://downloads.1password.com/linux/tar/stable/x86_64/1password-latest.tar.gz
+        sha256: 13b957cb1a012f6ce2308222b2c909261679ca62b2bafe335c4aa2814b54e352
```

Comment:
given that the url is `latest`, ie doesn't have a version, the sha256 WILL change and break. You should add `x-checker-data`. https://docs.flathub.org/docs/for-app-authors/requirements#external-data-checker-for-proprietary-applications

If there is a versioned URL it's better.

Same for the aarch64  binary.

Reply:
also can this be redistributed?

Reply:
Are you referring to 1Password's terms of service, whether it allows 1Password to be published in unauthorized places?

Reply:
yes that. Some frown upon redistribution which in that case be in violation.